Symfony/CS/Fixer/Symfony/OperatorsSpacesFixer.php

```diff
@@ -76,6 +76,6 @@ public function fix(\SplFileInfo $file, $content)
      */
     public function getDescription()
     {
-        return 'Binary operators should be surrounded by at least one space.';
```

Comment:
Really bad to have one fixer changing stuff, then another one reverting the changes.